apps/system/private_browser.html

```diff
@@ -2,7 +2,7 @@
 <head>
   <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1">
   <meta charset="utf-8">
-  <title data-l10n-id="private-window"></title>
+  <title>&nbsp;</title>
```

Comment:
Do we need the space, or can it just be an empty character? Is this to trick rocketbar?